app/assets/javascripts/components/campaign/campaign_overview_handler.jsx

```diff
@@ -1,8 +1,30 @@
 import React from 'react';
+import { useLocation } from 'react-router-dom';
+import CampaignStats from './campaign_stats';
 import CampaignStatsDownloadModal from './campaign_stats_download_modal';
+import CampaignNavbar from '../common/campaign_navbar';
 
 const CampaignOverviewHandler = (props) => {
-  return <CampaignStatsDownloadModal {...props} />;
+  const location = useLocation();
+  let statsModal;
+  if (location.pathname.match(/overview/)) {
+    statsModal = (
+      <div className="stats-download-modal">
+        <CampaignStatsDownloadModal {...props} />;
+      </div>);
+  }
+  let navBar;
+  if (location.pathname.match(/ores_plot/) || location.pathname.match(/alerts/)) {
```

Comment:
I don't like this. Rendering the CampaignNavbar (and the CampaignStats) should be the responsibility of the top `Campaign` component, rather than the overview handler.